server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -332,10 +332,11 @@
      * Logs an informational message indicating the server is starting.
      *
      * @param prettyVersion  the server version.
+     * @param banner the server ACII banner.
      */
     @LogMessage(level = INFO)
-    @Message(id = 49, value = "%s starting")
-    void serverStarting(String prettyVersion);
+    @Message(id = 49, value = "%s starting %n%s")
```

Comment:
Let's drop the %n and require it to be in the 'banner' text.

If we do this it needs to be complete no-op if JBoss-Project-Banner is not configured.

Reply:
That was the initial idea :)